README.md

```diff
@@ -67,6 +67,7 @@ Here is the supported scope list and the claims they contain. The details of eac
 * 'openid:government-gateway': government_gateway
 * 'openid:mdtp': mdtp
 * 'openid:gov-uk-identifiers': uk_gov_nino
+* 'openid:trusted-helper': trustedHelper
```

Comment:
My mistake, pls can we change this to "trusted_helper" to align with the naming convention of the other claims, thx